src/credentials/tests/TestChipCert.cpp

```diff
@@ -670,6 +830,11 @@ static const nlTest sTests[] = {
     NL_TEST_DEF("Test CHIP Certificate Validation time", TestChipCert_CertValidTime),
     NL_TEST_DEF("Test CHIP Certificate Usage", TestChipCert_CertUsage),
     NL_TEST_DEF("Test CHIP Certificate Type", TestChipCert_CertType),
+    NL_TEST_DEF("Test CHIP Generate Root Certificate", TestChipCert_GenerateRootCert),
```

Comment:
Non of these tests verifies that generated certificate is valid and it's signature is verifiable. Would be a good idea to add a test that generates all 3 certs Root --> ICA --> Node and doing full certificate chain validation

Reply:
Added one test that checks the 3 certs Root --> ICA --> Node chain. We can probably add more validations in the test. Maybe that can be done in a follow on PR.